WMFComponents/Sources/WMFComponents/Components/Suggested Edits/Alt Text Experiment/Preview/WMFAltTextExperimentPreviewViewController.swift

```diff
@@ -21,6 +21,7 @@ final public class WMFAltTextExperimentPreviewViewController: WMFCanvasViewContr
     private let hostingViewController: WMFAltTextExperimentPreviewHostingViewController
     private var viewModel: WMFAltTextExperimentPreviewViewModel
     public weak var delegate: WMFAltTextPreviewDelegate?
+    public var publishButton: UIBarButtonItem?
```

Comment:
(Minor) I would rather us not expose the button itself if we can avoid it. Can you make this property private, and make a public method that only toggles the publish button enabled state?

Reply:
Done!